examples/games/trex/pubspec.yaml

```diff
@@ -11,9 +11,9 @@ environment:
 
 dependencies:
   flame: ^1.1.1
-  google_fonts: ^2.3.2
   flutter:
     sdk: flutter
+  google_fonts: ^2.3.2
```

Comment:
Not need to do now, but I believe this isn't used anymore? I believe Pasha has changed this to use Sprite fonts.

Reply:
Removed